drydock_backups/docker-entrypoint.sh

```diff
@@ -25,13 +25,13 @@ else
     exit 1
 fi
 
-if [ "$BACKUPS_STORAGE_SERVICE" = 'aws-s3' ]; then
+if [ "$BACKUP_STORAGE_SERVICE" = 'aws-s3' ]; then
       if [[ -z "${BACKUP_CUSTOM_STORAGE_ENDPOINT}" ]]; then
             aws s3 mv $FILENAME s3://$S3_BUCKET_NAME/$BUCKET_PATH/$1/
       else
             aws --endpoint-url $BACKUP_CUSTOM_STORAGE_ENDPOINT s3 mv $FILENAME s3://$S3_BUCKET_NAME/$BUCKET_PATH/$1/
       fi
-elif [ "$BACKUPS_STORAGE_SERVICE" = 'azure-blob' ]; then
+elif [ "$BACKUP_STORAGE_SERVICE" = 'azure-blob' ]; then
```

Comment:
Does the backup image have installed `azcopy` already?

Reply:
I don't understand what you're referring to; however, [here](https://github.com/eduNEXT/drydock/blob/mar/az-backups/drydock_backups/Dockerfile#L14-L18) AzCopy is installed in the image. I built one and [used it](https://hub.docker.com/layers/ednxops/shipyard-utils/az-test/images/sha256-41e4ebafecfffc98a7b886223b3fddf5aaef6e02ae10d8594011630d24589b4f?context=explore) for testing

Reply:
Yes, that's what I was referring to.